Design.md

```diff
@@ -0,0 +1,52 @@
+# Design
+ 
+## Senior UI/UX Designer
+- I have at least 5 years' experience
+- I display initiative and ownership
+
+- I can lead a project day to day (Design Lead)
+    - I can make strategic design and UX decisions about both core and new functions and features
+    - I can measure and capture user feedback on a regular basis.
+- I can create design guidelines to set the tone and design rules through out a project life cycle
+- I can oversee the user experience from conception to launch in partnership with Product Owners and developers
+- I am responsible for overall team performance and building a culture of design excellence
+- I am comfortable with running the Scrum ceremonies (i.e. can act as Scrum Master)
+- I am comfortable with mentoring/teaching other designers 
+- I can define the user experience best practice for client porjects and internal projects
+
+
+##  UI/UX Designer
+- I have at least 3 years' experience
```

Comment:
1. Similar to above, we should replace "at least 3 years' experience" with a range e.g. 1-3 years' experience.

Reply:
Done